test/integration/aaa_download_only_test.go

```diff
@@ -158,3 +161,49 @@ func TestDownloadOnly(t *testing.T) {
 	})
 
 }
+
+func TestDownloadOnlyDocker(t *testing.T) {
```

Comment:
maybe limit this test only to docker driver ?

Reply:
wouldn't hut for others to test it either but main goal of this test is really caching for other tests